scripts/libmetal.mk

```diff
@@ -108,6 +108,9 @@ $(METAL_LIB_DIR)/stamp: $(METAL_BUILD_DIR)/Makefile
 
 $(METAL_LIB_DIR)/lib%.a: $(METAL_LIB_DIR)/stamp ;@:
 
+$(METAL_LIB_DIR)/libmetal-pico.a: $(METAL_LIB_DIR)/libriscv__pico__metal.a
```

Comment:
If we name the library what we want in freedom-metal, we can avoid defining a rule to rename it.

Reply:
yeah, I'm not sure why I had this funky name; I'll try to switch it.